pennylane/qnodes/jacobian.py

```diff
@@ -23,25 +23,47 @@
 
 from .base import BaseQNode, QuantumFunctionError
 
+DEFAULT_STEP_SIZE = 0.3
+DEFAULT_STEP_SIZE_ANALYTIC = 1e-7
+
 
 class JacobianQNode(BaseQNode):
     """Quantum node that can be differentiated with respect to its positional parameters.
     """
 
-    def __init__(self, func, device, mutable=True, properties=None):
-        super().__init__(func, device, mutable=mutable, properties=properties)
+    def __init__(self, func, device, mutable=True, **kwargs):
+        super().__init__(func, device, mutable=mutable, **kwargs)
 
         self.par_to_grad_method = None
         """dict[int, str]: map from flattened quantum function positional parameter index
         to the gradient method to be used with that parameter"""
 
+        try:
+            analytic = self.device.analytic
+        except AttributeError:
+            # The analytic attribute is not defined for hardware devices
+            analytic = False
```

Comment:
Nice!

Reply:
Nitpick: this could also be written
```python
analytic = getattr(self.device, "analytic", False)
```
This is similar to the `get` method of dictionaries --- if the attribute does not exist, the default value (False) is returned.

Marked as a nitpick because try-except is also very Pythonic, so up to you whether you want to change this!

Reply:
I think this looks better, thanks so much for it! :blush: Also included a better-formatted attribute docstring